src/style.css

```
@@ -0,0 +1,29 @@
+html {
+  background-color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.box_blue {
+  box-sizing: border-box;
+  background-color: #308dc0;
+  width: 350px;
+  border: #e95d49 solid 1px;
+  margin: 20px;
+  margin-bottom: 40px;
```

wrong way

margin: top right bottom left
```suggestion
  margin: 20px 20px 40px;
```